docs/public/static/schemas/unversioned/eas-json-build-common-schema.js

```diff
@@ -48,6 +48,13 @@ export default [
       '[Learn more about custom development clients](../../clients/introduction).',
     ],
   },
+  {
+    name: 'resourceClass',
+    enum: ['default', 'medium'],
```

Comment:
if we have a medium class, why there is a default too? aren't those the same?

Reply:
@wkozyra95 We discussed it here https://exponent-internal.slack.com/archives/C02123T524U/p1673430449218979?thread_ts=1673429656.875129&cid=C02123T524U

Reply:
Currently they both point to the medium instances, however default can be moved to large instances in the future, but medium will always explicitly point to medium instances